server/src/lib.rs

```diff
@@ -63,10 +63,7 @@ static JWTSECRET: Lazy<Vec<u8>> = Lazy::new(|| {
     hex::decode(jwt).expect("JWT secret should be a hex string")
 });
 
-pub async fn run() {
-    // TODO: think about channel size bound
-    let (state_channel, rx) = mpsc::channel(1_000);
-
+pub async fn run(max_buffered_commands: u32, max_concurrent_queries: u32) {
```

Comment:
We should keep the `max_buffered_commands` and `max_concurrent_queries` parameters in `GenesisConfig`.